onnxruntime/test/perftest/README.md

```diff
@@ -1,25 +1,63 @@
 # ONNXRuntime Performance Test
 
-onnxruntime_perf_test [options...] model_path result_file
+This tool provides the performance results using the ONNX Runtime with the specific execution provider to run the inference for a given model using the sample input test data. This tool can provide a reliable measurement for the inference latency usign ONNX Runtime on the HW platform. The options to use with the tool are listed below:
```

Comment:
HW is not How about change the HW platform to current platform.

Reply:
or ... on the __device__?